lib/IrcHandlers/IrcActionHandler.js

```diff
@@ -0,0 +1,50 @@
+const Helpers = require("../Helpers.js");
+
+/**
+ * Handles the event when a user performs an action in the IRC channel.
```

Comment:
By "performs an action" I assume you mean a `/me` command? For posterity, we might want to clarify that here, since "performing an action" is pretty ambiguous.

Reply:
Added a comment.